### PR TITLE
Reflects input changes to CBR

### PR DESCRIPTION
**Pull request by Vismayak**

### CODE REVIEW

**Comment by longshuicy** [comment]
You might also want to update the workshop session 3 and 4 notebook to accept this new input in the PR? If it's not too much to do

File: notebooks/commercial_building_recovery.ipynb [hunk 1]
```diff
@@ -87,7 +125,8 @@
     "# Load input datasets\n",
     "com_recovery.load_remote_input_dataset(\"sample_damage_states\", sample_damage_states)\n",
     "com_recovery.load_remote_input_dataset(\"mcs_failure\", mcs_failure)\n",
```
Comment (longshuicy) [inline notebooks/commercial_building_recovery.ipynb:127]: You might want to properly chain it with MCS so it doesn't confuse the user
```
Damage analysis --> MCS
 \               /
   \            /
    \         / 
    both into CBR
```

**Review by longshuicy — CHANGES_REQUESTED** [review]

**Comment by longshuicy** [comment]
> @longshuicy I already did update the workshop notebooks, thats the one where I pushed to main by mistake

That workshop notebook repos are private. Ya-Lan and Chris copied over the material to incore-docs so we can share it as a part of the documentation. Hence you might want to update it here. 

They should be under the path of `/manual_jb/content/workshops/20231115`. 

- You need to modify session 3 & session 4. Session 4 you can just add 
```
commercial_recovery.set_input_dataset("building_dmg", bldg_dmg_result)
```
- Session 3 I haven't tried. 
- Lastly, when notebooks updated, you would need to creating a zip file of them, and place it under `/workshops/20231115` to replace the outdated one. This is the place where we allow user to download the zipfile.

**Comment by ylyangtw** [comment]
Tested session3 in zip file, it works. session 4 changes also look good.

Could you remove the __MACOSX in the zip file? 
image
If you simply use macos default to zip it, __MACOSX will be generated automatically. You will want to use command line in the terminal instead.

**Review by navarroc — APPROVED** [review]

**Pull request merged**